auth.go

```diff
@@ -422,18 +419,23 @@ func setGroup(icentral *imqsauth.ImqsCentral, groupName string, roles []string)
 	return modifyGroup(icentral, groupModifySet, groupName, perms)
 }
 
-func createUser(icentral *imqsauth.ImqsCentral, options map[string]string, identity string, password string) bool {
-	if e := icentral.Central.CreateAuthenticatorIdentity(identity, password); e == nil {
-		fmt.Printf("Created user %v\n", identity)
+func createUser(icentral *imqsauth.ImqsCentral, options map[string]string, email string, username, firstname, lastname, mobilenumber, password string) bool {
```

Comment:
Should we not rather make the non-essential attributes optional?
The nonessentials are: firstname, lastname, mobile.
It's just extra typing for people setting stuff up on servers, where often one wouldn't even have the real info for these things (eg you're adding IMQS admin user who isn't really associated with a real person).